voter/src/main/java/org/terracotta/voter/ClusterState.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *
+ *  The contents of this file are subject to the Terracotta Public License Version
+ *  2.0 (the "License"); You may not use this file except in compliance with the
+ *  License. You may obtain a copy of the License at
+ *
+ *  http://terracotta.org/legal/terracotta-public-license.
+ *
+ *  Software distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
+ *  the specific language governing rights and limitations under the License.
+ *
+ *  The Covered Software is Terracotta Core.
+ *
+ *  The Initial Developer of the Covered Software is
+ *  Terracotta, Inc., a Software AG company
+ *
+ */
+package org.terracotta.voter;
+
+import com.terracotta.connection.api.DiagnosticConnectionService;
+import com.terracotta.diagnostic.Diagnostics;
+import java.net.URI;
+import java.util.Properties;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ */
+public class ClusterState {
```

Comment:
To be removed in the final PR.

Reply:
Are you opposed?  I thought it useful as an internal tool

Reply:
Fine with keeping it for internal use.